util/config/compileline

```diff
@@ -37,6 +46,15 @@ $make = "$make --no-print-directory";
 
 sub mysystem {
   my $cmd = shift;
+
+  # Uncomment for debug output
```

Comment:
I'm not sure yet... I needed it to do this. How about we turn it in to a flag if we need it again?